mayland-blocks/functions.php

```diff
@@ -3,21 +3,6 @@
 if ( ! function_exists( 'mayland_blocks_support' ) ) :
 	function mayland_blocks_support() {
 
-		// Alignwide and alignfull classes in the block editor.
-		add_theme_support( 'align-wide' );
-
-		// Adding support for core block visual styles.
-		add_theme_support( 'wp-block-styles' );
```

Comment:
Are you sure we don't want to keep this one in?

Reply:
My take is to be done with wp-block-styles forever.  Anything that support might style seems like it should be in theme.json instead so that it's explicit and understood WHY a block is styled as it is.